# HG changeset patch
# User Florian Pose
# Date 1540300607 -7200
#      Tue Oct 23 15:16:47 2018 +0200
Minor fix.
(grafted from b64492b5c10b318c2887245c6020970ac538dcc0)

diff --git a/devices/igb/update.sh b/devices/igb/update.sh
--- a/devices/igb/update.sh
+++ b/devices/igb/update.sh
@@ -1,15 +1,14 @@
 #!/bin/bash
 
-
-KERNELDIR=$1
-PREVER=$2
-KERNELVER=$3
-
 if [ $# -ne 3 ]; then
     echo "Need 3 arguments: 1) kernel source dir, 2) previous version, 3) version to add"
     exit 1
 fi
 
+KERNELDIR=$1
+PREVER=$2
+KERNELVER=$3
+
 IGBDIR=drivers/net/ethernet/intel/igb
 
 FILES="e1000_82575.c e1000_82575.h e1000_defines.h e1000_hw.h e1000_i210.c e1000_i210.h e1000_mac.c e1000_mac.h e1000_mbx.c e1000_mbx.h e1000_nvm.c e1000_nvm.h e1000_phy.c e1000_phy.h e1000_regs.h igb_ethtool.c igb.h igb_hwmon.c igb_main.c igb_ptp.c"
